doc/formatting.md

```diff
@@ -42,19 +43,6 @@ Sample help command line
                                     formatted.
 ```
 
-## Dependencies
-
-* clang-format - The jit-format tool calls clang-format to run whitespace based formatting changes.
-* clang-tidy - The jit-format tool calls clang-tidy to run more advanced formatting changes.
-
-Running `bootstrap.cmd` or `bootstrap.sh` will download the correct version of clang-format and clang-tidy
-for your platform, if the correct version is not already on your path.
-
-We require version 3.8.0 for jit-format. If you wish to install it yourself,
```

Comment:
> bootstrap doesn't re-download unless it can't find clang-format/clang-tidy

What does it do when we update the clang-format/clang-tidy in Azure to a newer version?